From 8df7aa00ff73ba199d1e2695c5cf99b0079fae3f Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Tue, 26 Jan 2010 11:55:28 +0100
Subject: [PATCH] Add AutostartCondition to gnome-at-session.desktop

The gnome-at-session autostart desktop file launches a shell script, which in
turn calls gconftool-2, just to find out that it's disabled, and exit. This
wastes unnecessary cycles on starting the desktop.

Add an AutostartCondition to the .desktop file which only runs when a11y is
enabled.

Bug: https://bugzilla.gnome.org/show_bug.cgi?id=608138
---
 .../gnome-at-session.desktop.in.in                 |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/capplets/default-applications/gnome-at-session.desktop.in.in b/capplets/default-applications/gnome-at-session.desktop.in.in
index 9480ad9..9c1e24b 100644
--- a/capplets/default-applications/gnome-at-session.desktop.in.in
+++ b/capplets/default-applications/gnome-at-session.desktop.in.in
@@ -7,6 +7,7 @@ Terminal=false
 Type=Application
 StartupNotify=false
 OnlyShowIn=GNOME;
+AutostartCondition=GNOME /desktop/gnome/interface/accessibility
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-control-center
 X-GNOME-Bugzilla-Component=other capplets
-- 
1.6.5

